packages/ui/src/components/SwipableChatListItem.tsx

```diff
@@ -81,7 +65,8 @@ export function SwipableChatRow(
           drag={drag}
           model={props.model}
           handleAction={handleAction}
-          jailBroken={props.jailBroken ?? false}
+          isMuted={mutedState}
+          // jailBroken={props.jailBroken ?? false}
```

Comment:
?

Reply:
I swear it was a cute feature flag once upon a time...